tests/feature_tests/feature_test_vars.sh

```diff
@@ -2,5 +2,10 @@
 
 export NUM_OF_CONTROLPLANE_REPLICAS="3"
 export NUM_OF_WORKER_REPLICAS="1"
-export FROM_K8S_VERSION="v1.24.0"
-export KUBERNETES_VERSION=${FROM_K8S_VERSION}
\ No newline at end of file
+if [ "${CAPM3RELEASEBRANCH}" == "release-0.5" ];
+then
+  export FROM_K8S_VERSION="v1.23.8"
```

Comment:
```suggestion
  export FROM_K8S_VERSION="v1.23.5"
```

Reply:
I have pushed the fix, page was just probably slow to update.